backend/YouTubeV2.Application/Services/VideoServices/VideoService.cs

```diff
@@ -32,5 +35,18 @@ public async Task<Guid> AddVideoMetadataAsync(VideoMetadataPostDTO videoMetadata
             await _blobImageService.UploadVideoThumbnailAsync(videoMetadata.thumbnail, video.Id.ToString(), cancellationToken);
             return video.Id;
         }
+
+        public async Task<Video?> GetVideoByIdAsync(Guid id, CancellationToken cancellationToken = default, params Expression<Func<Video, object>>[] includes)
```

Comment:
If this method is as simple as getting video by id then i dont see why it's so complicated. and if it is more complicated then change its name